formats/application/vnd/oada/isoblue/1/+json/examples/default.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  _type: 'application/vnd.oada.isoblue.1+json',
+
+  context: {},
```

Comment:
Context is also an old thing, doesn't belong here.

Reply:
Fixed in 5696ced